src/components/info-list.jsx

```diff
@@ -1,23 +1,29 @@
-import React from 'react';
+import React, { Fragment } from 'react';
 import PropTypes from 'prop-types';
 
 import '../styles/components/info-list.scss';
 
-const InfoList = ({ infoData, columns }) => (
-  <div className={`info-list${columns ? ' info-list--columns' : ''}`}>
-    {infoData.map(
-      // Only draw if there is content
-      item =>
-        item.content && (
-          <div className="info-list__item" key={item.title}>
-            <div className="info-list__item__title">{item.title}</div>
-            <div className="info-list__item__content">{item.content}</div>
-          </div>
-        ),
-    )}
-  </div>
-);
-
+const InfoList = ({ infoData, columns, isFlat }) =>
+  (!isFlat ? (
```

Comment:
Being pedantic here because your logic is absolutely correct: am wondering if it would be slightly clearer to condition on `isFlat` rather than `!isFlat`?